pkg/agent/dozer/bcm/plan.go

```diff
@@ -1225,6 +1245,14 @@ func planMCLAGDomain(agent *agentapi.Agent, spec *dozer.Spec) (bool, error) {
 		IPv4Unicast: pointer.To(true),
 	}
 
+	spec.LSTGroups[LSTGroupSpineLink] = &dozer.SpecLSTGroup{
+		AllEVPNESDownstream: pointer.To(false),
```

Comment:
I believe just skipping it (it'll be nil) is a bit better - will not lead to agent trying to reconfigure. Just check the diff on agent trying to apply that